disa_app/lib/view_data_entrant_manager.py

```diff
@@ -56,7 +56,9 @@ def prep_get_response( self, rfrnt: models_alch.Referent ) -> dict:
             { 'first': n.first, 'last': n.last,
                 'name_type': n.name_type.name,
                 'id': n.id } for n in rfrnt.names ]
-        data['ent']['age'] = rfrnt.age
+        data['ent']['age_text'] = rfrnt.age
+        data['ent']['age_number'] = rfrnt.age_number
```

Comment:
@prashleigh I am currently confused about where to use age_integer versus age_number.